docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -192,18 +192,18 @@ The following example formats a <xref:System.Double> and an <xref:System.Int32>
 
 The general ("G") format specifier converts a number to the more compact of either fixed-point or scientific notation, depending on the type of the number and whether a precision specifier is present. The precision specifier defines the maximum number of significant digits that can appear in the result string. If the precision specifier is omitted or zero, the type of the number determines the default precision, as indicated in the following table.
 
-|Numeric type|Default precision|
-|------------------|-----------------------|
-|<xref:System.Byte> or <xref:System.SByte>|3 digits|
+|Numeric type                               | Default precision |
+|-------------------------------------------|-------------------|
+|<xref:System.Byte> or <xref:System.SByte>  |3 digits|
 |<xref:System.Int16> or <xref:System.UInt16>|5 digits|
 |<xref:System.Int32> or <xref:System.UInt32>|10 digits|
-|<xref:System.Int64>|19 digits|
-|<xref:System.UInt64>|20 digits|
-|<xref:System.Numerics.BigInteger>|Unlimited (same as ["R"](#RFormatString))|
-|<xref:System.Half>|3 digits|
-|<xref:System.Single>|7 digits|
-|<xref:System.Double>|15 digits|
-|<xref:System.Decimal>|29 digits|
+|<xref:System.Int64>                        |19 digits|
+|<xref:System.UInt64>                       |20 digits|
+|<xref:System.Numerics.BigInteger>          |Unlimited (same as ["R"](#RFormatString))|
+|<xref:System.Half>                         |3 digits|
+|<xref:System.Single>                       |9 digits (7 digits in .NET Framework)|
+|<xref:System.Double>                       |17 digits (15 digits in .NET Framework)|
```

Comment:
`Half` would be 5, not `3`. But `5`, `9`, and `17` aren't really "accurate" here at all.

For the floating-point types, we print the "shortest roundtrippable string" (equivalent to `R`) and so even if more digits of precision are available, we may not surface them if there is a shorter string that also equates to the same value.

Reply:
@tannergooding  Hmm. So are you saying there is no default precision value for the types when omitted? That's what the table is explaining, the default precision.

Reply:
For Half, Single, and Double on .NET Core 3.1 and later, that's roughly correct.

They don't have a precision and `G5`/`G9`/`G17` will all turn a distinct result from what `G` does. The same is true for `decimal` going back to .NET Framework.

Reply:
@tannergooding 

I think that is what the paragraphs after the table explain:

> When used with a <xref:System.Double> value, the "G17" format specifier ensures that the original <xref:System.Double> value successfully round-trips. This is because <xref:System.Double> is an IEEE 754-2008-compliant double-precision (`binary64`) floating-point number that gives up to 17 significant digits of precision. On .NET Framework, we recommend its use instead of the ["R" format specifier](#RFormatString), since in some cases "R" fails to successfully round-trip double-precision floating point values.

Should we update this with more information that states single/decimal/float/double all return a value that round-trips if available?

Reply:
The issue with that paragraph is the "up to 17 significant digits" part.

The most digits you need to guarantee a roundtrippable value for `double` is 17. However, the actual value represented often has many more digits (up to 767 of them).

The same applies to `float` where the most you need is `9` but where the actual represented may have up to `112` significant digits and similarly for `Half` with `5` and up to `19`.